controller/setting_controller.go

```diff
@@ -626,31 +616,28 @@ func (sc *SettingController) updateNodeSelector() error {
 	return nil
 }
 
-func (bm *BackupStoreMonitor) Start() {
+func (bm *BackupStoreTimer) Start() {
```

Comment:
This function is responsible for blindly updating `backupTarget.spec.syncRequestedAt` only.